ign_ros2_control/src/ign_system.cpp

```diff
@@ -426,18 +426,26 @@ hardware_interface::return_type IgnitionSystem::write()
     }
 
     if (this->dataPtr->joints_[i].joint_control_method & POSITION) {
-      if (!this->dataPtr->ecm->Component<ignition::gazebo::components::JointPositionReset>(
-          this->dataPtr->joints_[i].sim_joint))
-      {
+      // Get error in position
+      double error;
+      error = (this->dataPtr->joints_[i].joint_position -
+        this->dataPtr->joints_[i].joint_position_cmd ) * 100;
```

Comment:
why are you using here `* 100`. Shouldn't be this calculated with the time step to get velocity? I don't really get the idea behind this.

Reply:
I included this value because for some reason the controller was not reaching the subgoals

I need to spend more time on this